static/js/components/ProgressWidget_test.js

```diff
@@ -41,11 +41,21 @@ export const program = {
           "position": 1,
           "title": "Gio Test Course #14",
           "course_id": "course-v1:odl+GIO101+FALL14",
-          "status": STATUS_NOT_PASSED,
+          "status": STATUS_PASSED,
           "id": 2,
           "course_start_date": "2016-08-22T11:48:27Z",
-          "fuzzy_start_date": "Fall 2017",
+          "fuzzy_start_date": "Fall 2016",
           "course_end_date": "2016-09-09T10:20:10Z"
+        },
+        {
+          "position": 2,
```

Comment:
You've added a second run of this course, but the first is passing and the second is not. Based on the way the code used to be, this automated test would pass either way.

Can you modify another one of the courses in this program to have multiple runs, and set the status of the first course run to `STATUS_NOT_PASSED`? That would be an appropriate test of this change.